tests/acl/test_acl.py

```diff
@@ -278,7 +306,7 @@ def create_or_remove_acl_table(duthost, acl_table_config, setup, op):
                 )
             )
         else:
-            logger.info("Removing ACL table \"{}\" in namesspace {}".format(acl_table_config["table_name"], namespace))
+            logger.info("Removing ACL table \"{}\" in namespace {}".format(acl_table_config["table_name"], namespace))
```

Comment:
Minor: Device name can be added here as well, just like how it is added in creating ACL table.